### PR TITLE
CPA-29 Display nested comments

### PR DESCRIPTION
**Pull request by KenAdams777**
Implemented a new feature. Also fixed bugs and added a custom hook for more convenience using date format.
Feel free to leave your comments and suggestions.

### CODE REVIEW

File: src/components/Comment/Comment.js [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
-				</div>
-				<div className="Comment__mainInfo-container">
-					<div className="Comment__mainInfo-authorName">
-						{comment.author.fullname}
+			<div className="Comment__container">
+				<div className="Comment__body">{comment.comment}</div>
+				<div className="Comment__footer">
+					<div className="Comment__avatar-container">
+						<img
+							src={comment.author.avatar || noUserImageIcon}
```
Comment (bejalane) [inline src/components/Comment/Comment.js:16]: I would use optional chaining here as well.. u never know what you can get from BE

File: src/components/Poem/Poem.js [hunk 2]
```diff
@@ -164,16 +159,19 @@ Poem.ReadMoreButton = function PoemReadMoreButton({ ...restProps }) {
 };
 
 Poem.TagsList = function PoemTagslist({ poem, children, ...restProps }) {
-	const arrayToShow = Array.isArray(poem?.tags) ? poem.tags : poem.tags.split(',');
+	const arrayToShow = Array.isArray(poem?.tags)
```
Comment (bejalane) [inline src/components/Poem/Poem.js:162]: arrayToShow is a bad naming, you need to change to tagsToShow at least or smth more meaningful.

**Review by bejalane — APPROVED** [review]
Approved with minor comments

File: src/components/Poem/Poem.js (hunk 2 above)
Comment (KenAdams777) [reply src/components/Poem/Poem.js:162]: > arrayToShow is a bad naming, you need to change to tagsToShow at least or smth more meaningful.

This is not my feature. All questions to @RudeMax  )

File: src/services/utils/hooks.js [hunk 3]
```diff
@@ -8,3 +10,14 @@ export const useActions = () => {
...
 };
+
+export const useDate = (date) => {
+	const dateToDisplay =
+		new Date().getTime() - new Date(date).getTime() < 6.048e8;
+
+	return (
+		<Moment fromNow={dateToDisplay} calendar={dateFormatMoreThenWeek}>
+			{date}
+		</Moment>
```
Comment (yuriy1901) [inline src/services/utils/hooks.js:21]: for what did you did hook for date format? We already have function in services/utils/renderDataFormat

Comment (KenAdams777) [reply src/services/utils/hooks.js:21]: > for what did you did hook for date format? We already have function in services/utils/renderDataFormat

I did so in order do not duplicate code. Yes we had funciont but still when you need to display date anywhere else you need to use same code:
`<Moment fromNow={dateToDisplay} calendar={dateFormatMoreThenWeek}>
{date} 
</Moment>`.

**Review by yuriy1901 — APPROVED** [review]

**Pull request merged**